code/datums/outfits/jobs/job.dm

```diff
@@ -13,8 +13,13 @@
 
 	flags = OUTFIT_HAS_BACKPACK
 
-/decl/hierarchy/outfit/job/equip_id(mob/living/carbon/human/H)
+//VOREStation Edit START
+/decl/hierarchy/outfit/job/equip_id(mob/living/carbon/human/H, rank, assignment)
 	var/obj/item/weapon/card/id/C = ..()
+	var/datum/job/J = job_master.GetJob(rank)
+	if(J)
+		C.access = J.get_access()
+//VOREStation Edit END - fixes outfit system not giving centcom officers access on their id
```

Comment:
This bit of code kind of makes me nervous, because I am not sure every instance of equip_id has rank and assignment input into it. If it does work, this should probably be pushed to Polaris.

Also, CCO never had ID issues to begin with to my knowledge?

Reply:
CCOs don't have ID issues in the current code because they're still using the spawnId method from before the outfit system, rather than equip_id.  Giving them outfits changes that, I noticed that while testing the other changes.  It probably should be pushed to Polaris, yeah, I've been working on a refactor that includes it, submitted the pull request for that now over there.